EvidenceApi.Tests/V1/UseCase/CreateDocumentSubmissionUseCaseTests.cs

```diff
@@ -68,12 +69,12 @@ public void ThrowsNotFoundExceptionWhenEvidenceRequestIsNull()
                 .Setup(x => x.CreateDocumentSubmission(It.Is<DocumentSubmission>(x => x.DocumentTypeId == request.DocumentType)))
                 .Returns(() => null)
                 .Verifiable();
-            Func<DocumentSubmissionResponse> testDelegate = () => _classUnderTest.Execute(Guid.NewGuid(), request);
+            Func<Task<DocumentSubmissionResponse>> testDelegate = async () => await _classUnderTest.ExecuteAsync(Guid.NewGuid(), request).ConfigureAwait(true);
             testDelegate.Should().Throw<NotFoundException>();
         }
 
         [Test]
-        public void ReturnsTheCreatedDocumentSubmissionWhenRequestIsValid()
+        public async Task ReturnsTheCreatedDocumentSubmissionWhenRequestIsValid()
```

Comment:
This test is huge, so is quite difficult to follow. Would be a good candidate for a refactor later.